CONTRIBUTING.md

```diff
@@ -1,22 +1,53 @@
 # Contributing to Git Extensions
 
-## How to contribute code
+GitExtensions is a project with a long history, made possible by [hundreds of contributors](https://github.com/gitextensions/gitextensions/graphs/contributors).
 
-* Login in github (you need an account)
-* Fork the main repository from [github](http://github.com/gitextensions/gitextensions)
-* Read [Project workflow](https://github.com/gitextensions/gitextensions/wiki/Project-Workflow) and [Coding guide](https://github.com/gitextensions/gitextensions/wiki#coding-guide)
-* Push your changes to your fork
-* Create a pull request
+We welcome contributions including:
 
-## How to debug GitExtensions
+- Ideas for improvements
+- Bug reports
+- Bug fixes
+- New features
 
-The installer is build using WiX. You need to install WiX when you want to build the installer. This can be downloaded here: [http://wixtoolset.org/](http://wixtoolset.org/). If you do not want to build the installer, just open the solution and ignore the warning.
+To help the project maintainers be as effective as possible, please follow some simple guidelines.
 
-* Open the solution file (GitExtensions.sln)
-* Hit F5 to compile and run GitExtensions
+## Reporting Issues
 
-## How to create the installer
+[Search the issue tracker](https://github.com/gitextensions/gitextensions/issues?&q=) for an
+existing or closely related issue before creating a new one. Be sure to include closed issues
+in your search.
 
-* Download and install WiX [http://wixtoolset.org/](http://wixtoolset.org/)
-* Run Setup\\BuildInstallers.cmd to build the installers
+If an issue already exists, read through the discussion. If you can something helpful, do so.
+You can vote on issues that you'd like to see prioritised.
 
+If no issue exists, create one. Complete the template, and add any further information that
+could be relevant.
+
+## Pull Requests
+
+Want to contribute some code? Great! In addition to the regular GitHub Pull Request workflow,
+you'll want to browse the [wiki](https://github.com/gitextensions/gitextensions/wiki).
```

Comment:
`Wiki`

Reply:
It's not a proper noun :)

https://en.wikipedia.org/wiki/Wiki